dask_expr/expr.py

```diff
@@ -1029,23 +1041,27 @@ class AsType(Elemwise):
 
 
 class IsNa(Elemwise):
+    projection_passthrough = True
     _parameters = ["frame"]
     operation = M.isna
 
 
 class Round(Elemwise):
+    projection_passthrough = True
     _parameters = ["frame", "decimals"]
     operation = M.round
 
 
 class Abs(Elemwise):
+    projection_passthrough = True
     _parameters = ["frame"]
     operation = M.abs
 
 
 class Apply(Elemwise):
     """A good example of writing a less-trivial blockwise operation"""
 
+    projection_passthrough = True
```

Comment:
This one seems suspect to me

Reply:
Why? Both of them are column-wise as long as we don't support an axis keyword. So we can drop columns that aren't needed. 

Or is your concern with regards to side-effects of potential udfs?

Reply:
I have to correct myself: You can do whatever you want if your udf is given as a string (drop for example). I don't know why you would use it like that, but this means that we can't push projections through.